manifests/dht.toml

```diff
@@ -1,62 +1,81 @@
+[[base]]
 name = "dht"
-# hashicorp/go-getter URLs, so in the future we can support fetching test plans
-# from GitHub.
+# hashicorp/go-getter URLs, so in the future we fetch plans from anywhere
 source_path = "file:${TESTGROUND_SRCDIR}/plans/dht"
+    [defaults]
+    provisioner = "terraform:aws"
```

Comment:
I don't think Terraform should appear here. We are certainly not going to run a terraform script from the testground; IMO that's too much magic / "spooky action at a distance". We should just assume that if the user is hitting an AWS backend, they've already prepared it accordingly (with Terraform scripts).

What we can do is print out of a warning with instructions if we detect that the environment is not properly set up, e.g. if the docker swarm commands fail. The warning could look like this:

```
WARNING

Testground has detected that your AWS environment is not properly set up.
Follow the instructions here to fix this: git.io/...
```

Reply:
In the same vein, I don't think we should be calling this a provisioner, but rather a backend. We are not provisioning any machines, we are just using an infrastructure backend to run our tests.

Reply:
Gotcha. So for the `test run` itself this would be just the `backend` or `infra`.

`Testground` can still offer a command for `provision`, no? Or do we want to keep all the infra scripts in a folder forever like it is coming up on #111